beacon_node/http_api/src/lib.rs

```diff
@@ -3293,7 +3293,13 @@ pub fn serve<T: BeaconChainTypes>(
                         match chain.verify_aggregated_attestation_for_gossip(aggregate) {
                             Ok(verified_aggregate) => {
                                 messages.push(PubsubMessage::AggregateAndProofAttestation(Box::new(
-                                    verified_aggregate.aggregate().clone(),
+                                    aggregate.lazy().map_err(|e| {
+                            warp_utils::reject::custom_bad_request(format!(
+                                "unable to decode as LazySignedAggregateAndProof: {:?}",
```

Comment:
rustfmt seems to be struggling here, sometimes it helps to break up the string with a `\` so it doesn't get pushed past the 100th column